.github/workflows/main.yml

```diff
@@ -0,0 +1,73 @@
+name: CI
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  js_build_and_test:
+    name: JS Build and Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '10'
+      - run: npm ci
+      - run: npm run eslint
+      - run: npm run test
+  php_build_and_test:
+    name: PHP Build and Test
+    runs-on: ubuntu-latest
+    services:
+      mariadb:
+        image: mariadb:latest
+        ports:
+          - 3306:3306
+        env:
+          MYSQL_USER: wp_test
+          MYSQL_PASSWORD: wp_test
+          MYSQL_DATABASE: wordpress_default
+          MYSQL_ROOT_PASSWORD: root
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+    steps:
+      - name: Checkout latest WCS
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Switch to PHP 7.4
+        run: |
+          sudo update-alternatives --set php /usr/bin/php7.4
+          sudo update-alternatives --set phar /usr/bin/phar7.4
+          sudo update-alternatives --set phar.phar /usr/bin/phar.phar7.4
+      - name: Verify PHP version
```

Comment:
I added some verifying steps to help debugging. These can be removed when we decided to use this in prod.